src/passes/Asyncify.cpp

```diff
@@ -1532,7 +1532,7 @@ struct Asyncify : public Pass {
     auto verbose =
       runner->options.getArgumentOrDefault("asyncify-verbose", "") != "";
     auto sideModule =
```

Comment:
Please also rename this local and other places it appears to `relocatable`.

Reply:
I'd forgot the local variable name. Thanks!